src/rpc_api/data_types.rs

```diff
@@ -119,6 +119,8 @@ pub struct MessageLookup {
     pub return_dec: IpldJson,
```

Comment:
```suggestion
    #[serde(with = "crate::lotus_json")]
    pub return_dec: Ipld,
```

Reply:
It's not really related to your changes, but it's a small improvement to the API.